addons/escoria-dialog-simple/types/floating.gd

```diff
@@ -102,4 +112,5 @@ func _on_dialog_finished():
 	# Make the speaking item animation stop talking, if it is still alive
 	if is_instance_valid(_current_character) and _current_character != null:
 		_current_character.stop_talking()
+	set_process(false)
```

Comment:
This isn't really required I think, because the dialog manager removes the node from the scene tree anyway.

See https://github.com/godot-escoria/escoria-demo-game/blob/develop/addons/escoria-dialog-simple/esc_dialog_simple.gd#L55

Reply:
Done